src/main/java/com/ifmo/epampractice/daoimpl/DrivingLicenseImpl.java

```diff
@@ -0,0 +1,113 @@
+    private static final String DELETE_QUERY = "DELETE FROM driving_license WHERE ID = ?";
+    private static final String SAVE_QUERY = "INSERT INTO driving_license (Issue_date, Expiration_date, Serial_number) VALUES (?, ?, ?)";
+
+    private static final Logger LOG = LogManager.getLogger(DrivingLicenseImpl.class);
+    private DBConnectorInterface dbConnector = DBConnectorPostgres.getInstance();
+
+    @Override
+    public List<DrivingLicenseEntity> getAll() {
+        List<DrivingLicenseEntity> drivingLicenseEntityArrayList = new ArrayList<>();
+
+        try (Connection connection = dbConnector.getConnection();
+             Statement statement = connection.createStatement(
+                     ResultSet.TYPE_FORWARD_ONLY, ResultSet.CONCUR_READ_ONLY);
+             ResultSet resultSet = statement.executeQuery(GET_ALL_QUERY)) {
+            while (resultSet.next()) {
+                drivingLicenseEntityArrayList.add(entityFromResultSet(resultSet));
+            }
+        } catch (SQLException e) {
+            LOG.error(e);
+        }
+        return drivingLicenseEntityArrayList;
+    }
+
+    @Override
+    public DrivingLicenseEntity get(int id) {
+        DBConnectorInterface dbConnector = DBConnectorPostgres.getInstance();
+        try (Connection connection = dbConnector.getConnection();
+             PreparedStatement statement = connection.prepareStatement(GET_QUERY,
+                     ResultSet.TYPE_FORWARD_ONLY, ResultSet.CONCUR_READ_ONLY)) {
+            statement.setInt(1, id);
+            try (ResultSet resultSet = statement.executeQuery()) {
+                if (resultSet.next()) {
+                    return entityFromResultSet(resultSet);
+                }
+            }
+        } catch (SQLException e) {
+            LOG.error(e);
+        }
+        return null;
+    }
+
+    @Override
+    public void update(DrivingLicenseEntity entity) {
+        DBConnectorInterface dbConnector = DBConnectorPostgres.getInstance();
+        try (Connection connection = dbConnector.getConnection();
+             PreparedStatement statement = connection.prepareStatement(UPDATE_QUERY)) {
+            setStatementFields(statement, entity);
+            statement.setInt(4, entity.getId());
+            statement.executeUpdate();
+        } catch (SQLException e) {
+            LOG.error(e);
+        }
+
+    }
+
+    @Override
+    public void delete(DrivingLicenseEntity entity) {
+        DBConnectorInterface dbConnector = DBConnectorPostgres.getInstance();
+        try (Connection connection = dbConnector.getConnection();
+             PreparedStatement statement = connection.prepareStatement(DELETE_QUERY)) {
+            statement.setInt(1, entity.getId());
+            statement.execute();
+        } catch (SQLException e) {
+            LOG.error(e);
+        }
+    }
+
+    @Override
+    public void save(DrivingLicenseEntity entity) {
+        DBConnectorInterface dbConnector = DBConnectorPostgres.getInstance();
```

Comment:
you declare dbConnector as class-private field (line 23) and then declare it again here